packages/api/src/graphql/comment-rating/comment-rating.public-mutation.ts

```diff
@@ -0,0 +1,54 @@
+import {PrismaClient, RatingSystemType} from '@prisma/client'
+import {Context} from '../../context'
+import {InvalidStarRatingValueError, NotFound} from '../../error'
+
+export const rateComment = async (
+  commentId: string,
+  answerId: string,
+  value: number,
+  fingerprint: string | undefined,
+  optionalAuthenticateUser: Context['optionalAuthenticateUser'],
+  commentRatingSystemAnswer: PrismaClient['commentRatingSystemAnswer'],
+  commentRating: PrismaClient['commentRating']
+) => {
+  const session = optionalAuthenticateUser()
+
+  const answer = await commentRatingSystemAnswer.findUnique({
+    where: {
+      id: answerId
+    }
+  })
+
+  if (!answer) {
+    throw new NotFound('CommentRatingSystemAnswer', answerId)
+  }
+
+  switch (answer.type) {
+    case RatingSystemType.star: {
+      if (value < 0 || value > 5) {
+        throw new InvalidStarRatingValueError()
+      }
+    }
```

Comment:
We should handle the default switch case. It should throw an error
```suggestion
    }
    default: {
        throw new NotFound(`Answer type ${answer.type} not available.`)
    }
```

Reply:
This should not happen as it's data we control. I would not check for values that one has to manually change the database.